releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,15 @@
 
 [Image][image] | [Sample][sample]
 
+## 1.2.0 - 
+
+### Enhancements
+* Added support for feature flags, which allows projecting feature flags to the target ConfigMap using [Microsoft Feature Flag schema](https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureManagement.v1.0.0.schema.json). The ConfigMap can be consumed by Feature Management libraries in all languages.

```

Comment:
This is not something that can be explained in a sentence or two. The support in other languages is indepent on this release. In this case, I would skip most of the details and have something like below. We can include a link to a document if we have one ready when this release note is published.

* Added support for downloading feature flags from Azure App Configuration.

Reply:
Not sure the "link to a doc" refers to what doc, and where the link to include.

Reply:
The quickstart for feature management using the k8s provider.